e2e_test/source_inline/kafka/temporary_kafka_batch.slt

```diff
@@ -71,6 +71,12 @@ t
 t
 t
 
+# ensure now() is larger
+sleep 1s
```

Comment:
The value of `now()` is updated if there's a new hummock snapshot synchronized to the frontend. By `FLUSH` we are sure there's a new snapshot generated, but the notification still remains asynchronous.

Reply:
🧐 I had thought after `FLUSH` we will definitely go to next epoch

Reply:
> I had thought after `FLUSH` we will definitely go to next epoch

Ah you're right. I'm a little dumb.

https://github.com/risingwavelabs/risingwave/blob/b3655662fd366845a99caaf2b3e95dbb5bbffbe1/src/frontend/src/handler/flush.rs#L31-L37